src/util/execute-lifecycle-script.js

```diff
@@ -45,7 +37,6 @@ export async function getWrappersFolder(config: Config): Promise<string> {
 
   await makePortableProxyScript(process.execPath, wrappersFolder, {
```

Comment:
Shall we unify this logic in the future and use `node` at all times?

Reply:
What do you mean? At most we should replace `makePortableScript` with `cmdShim` (which already covers the behavior we're using here, I think)